plugins/openvr/src/OpenVrHelpers.cpp

```diff
@@ -126,7 +126,7 @@ vr::IVRSystem* acquireOpenVrSystem() {
             #endif
 
             if (eError == 108) { // vr::HmdError_Init_HmdNotFound
-                headsetNotConnected = 1;
+                isHMDConnected = true;
```

Comment:
Now it doesn't make sense,   since it wont be connected if the error happens,  hehehehe, flip the bools or rename the bool to `isHMDInErrorState` or so (sorry it's getting late :) )